src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends ColorSupplier implements Figure {

```

Comment:
The Square class should not extend ColorSupplier. Instead, it should have a ColorSupplier instance if needed.